### PR TITLE
Move to a gunicorn + traefik setup

### PR DESCRIPTION
**Pull request by enolfc**
Make this more friendly to tools that can get automatically certs (traefik) without changing the container for every deployment.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      - "--providers.docker.exposedbydefault=false"
+      - "--entrypoints.web.address=:80"
+      - "--entrypoints.websecure.address=:443"
+      - "--entrypoints.web.http.redirections.entryPoint.to=websecure"
+      - "--entrypoints.web.http.redirections.entryPoint.scheme=https"
+      - "--entrypoints.web.http.redirections.entrypoint.permanent=true"
+      - "--certificatesresolvers.myresolver.acme.httpchallenge=true"
+      - "--certificatesresolvers.myresolver.acme.httpchallenge.entrypoint=web"
+      #- "--certificatesresolvers.myresolver.acme.caserver=https://acme-staging-v02.api.letsencrypt.org/directory"
+      - "--certificatesresolvers.myresolver.acme.email=enol.fernandez@egi.eu"
```
Comment (enolfc) [inline docker-compose.yaml]: hardcoded :(

Comment (enolfc) [reply docker-compose.yaml]:
```suggestion
      - "--certificatesresolvers.myresolver.acme.email=cloud-compute-support@mailman.egi.eu"
```

**Comment by enolfc** [comment]
This goes together with this other one: https://github.com/EGI-Federation/fedcloud-dashboard/pull/5

**Comment by sebastian-luna-valero** [comment]
Closing in favor of https://github.com/EGI-Federation/fedcloud-dashboard/pull/10

Many thanks @enolfc!

**Pull request closed without merging**